src/Migrations/2019/Version20190801093025.php

```diff
@@ -0,0 +1,146 @@
+<?php
```

Comment:
why would you need a migration for this ticket?
I think there went something wrong